snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/EvaluateQueryRefSetRequest.java

```diff
@@ -47,20 +47,9 @@ public final class EvaluateQueryRefSetRequest extends IndexResourceRequest<Branc
 	public QueryRefSetMemberEvaluations execute(final BranchContext context) {
 		final SnomedReferenceSet referenceSet = SnomedRequests.prepareGetReferenceSet(referenceSetId).build().execute(context);
 		return new QueryRefSetMemberEvaluations(
-				FluentIterable
-				.from(getQueryMembers(context, referenceSet))
-				.transform(new Function<SnomedReferenceSetMember, Request<BranchContext, QueryRefSetMemberEvaluation>>() {
-					@Override
-					public Request<BranchContext, QueryRefSetMemberEvaluation> apply(SnomedReferenceSetMember input) {
-						return SnomedRequests.prepareQueryRefSetMemberEvaluation(input.getId()).setLocales(locales()).setExpand(expand()).build();
-					}
-				})
-				.transform(new Function<Request<BranchContext, QueryRefSetMemberEvaluation>, QueryRefSetMemberEvaluation>() {
-					@Override
-					public QueryRefSetMemberEvaluation apply(Request<BranchContext, QueryRefSetMemberEvaluation> input) {
-						return input.execute(context);
-					}
-				})
+				getQueryMembers(context, referenceSet).stream()
+				.map(input -> SnomedRequests.prepareQueryRefSetMemberEvaluation(input.getId()).setLocales(locales()).setExpand(expand()).build())
+				.map(input -> input.execute(context))
```

Comment:
These two mapping steps could be merged together; is there a reason they are kept separate?